Lib/ufo2ft/fastUFO.py

```diff
@@ -155,19 +193,34 @@ def set_unicode(self, val):
 class Info(object):
     def __init__(self):
         attrs = (
-            'guidelines note openTypeGaspRangeRecords openTypeHeadFlags '
+            'familyName styleName '
+            'note openTypeGaspRangeRecords openTypeHeadFlags '
             'openTypeNameDescription openTypeNameLicense '
-            'openTypeNameLicenseURL openTypeNameRecords openTypeNameSampleText '
+            'openTypeNameLicenseURL openTypeNameRecords '
+            'openTypeNameSampleText '
             'openTypeNameUniqueID openTypeNameVersion '
             'openTypeNameWWSFamilyName openTypeNameWWSSubfamilyName '
             'openTypeOS2CodePageRanges openTypeOS2FamilyClass '
             'openTypeOS2Panose openTypeOS2Type openTypeOS2UnicodeRanges '
+            'openTypeOS2Selection openTypeHheaAscender openTypeHheaDescender '
+            'openTypeHheaLineGap openTypeOS2TypoAscender '
+            'openTypeOS2TypoDescender openTypeOS2TypoLineGap '
+            'openTypeOS2WinAscent openTypeOS2WinDescent '
+            'openTypeOS2WidthClass openTypeHeadCreated '
             'openTypeOS2VendorID postscriptDefaultCharacter '
             'postscriptForceBold postscriptIsFixedPitch '
-            'postscriptWindowsCharacterSet trademark unitsPerEm'
+            'postscriptWindowsCharacterSet trademark unitsPerEm '
+            'postscriptUnderlineThickness postscriptUnderlinePosition '
+            'openTypeOS2WidthClass openTypeOS2WeightClass'
         ).split()
         for attr in attrs:
             setattr(self, attr, None)
+        array_attrs = (
+            'guidelines postscriptFamilyBlues postscriptFamilyOtherBlues '
+            'postscriptStemSnapH postscriptStemSnapV'
+        ).split()
```

Comment:
I’m curious why not using a tuple directly here (and above), does it speed things or just a style choice?